packages/models-library/src/models_library/projects_networks.py

```diff
@@ -1,15 +1,22 @@
+import re
+
 from models_library.projects import ProjectID
-from pydantic import BaseModel, Field, constr
+from pydantic import BaseModel, ConstrainedStr, Field
 
 from .generics import DictModel
 from .projects_nodes_io import NodeIDStr
 
-SERVICE_NETWORK_RE = r"^[a-zA-Z]([a-zA-Z0-9_-]{0,63})$"
+SERVICE_NETWORK_RE = re.compile(r"^[a-zA-Z]([a-zA-Z0-9_-]{0,63})$")
```

Comment:
THOUGHT: It would be good to have a convention about these "regex" constants: i.e. is it a regext= 
string or a ``re.compile`` object?

We have been mixing it even in https://github.com/ITISFoundation/osparc-simcore/blob/d1522704f5674464eb81b90a5315c01110c0472e/packages/models-library/src/models_library/basic_regex.py

In any case please annotate to highlight the change
```python
SERVICE_NETWORK_RE: Final[re.Pattern] = re.compile
```